x-pack/plugins/ui_actions_enhanced/public/drilldowns/url_drilldown/url_drilldown_definition.tsx

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
I would split the action license support into its own dedicated PR. To ensure the new APIs are used (and tested), you could modify one of the examples, or add a new one.

Reply:
@stacey-gammon, oh, forgot to remove reviewers from this on. this is a cleaned up version of just license support: https://github.com/elastic/kibana/pull/68507